packages/slice-machine/lib/models/common/http/FakeClient.ts

```diff
@@ -8,7 +8,9 @@ export interface FakeResponse {
 }
 
 export default class FakeClient {
-  private mutate: (jsonResponse?: any) => {
+  private mutate: (
+    jsonResponse?: any
```

Comment:
same question here, I don't understand the rule that's applied here

Reply:
it's not a linting rule, it's prettier.... could have something to do with line length or method declarations (i'm not sure tbh)